src/main/scala/mesosphere/marathon/upgrade/StartingBehavior.scala

```diff
@@ -86,13 +55,14 @@ trait StartingBehavior { this: Actor with ActorLogging =>
       context.system.scheduler.scheduleOnce(5.seconds, self, Sync)
   }
 
+  override def taskIsReady(taskId: Id): Unit = {
+    log.info(s"New task $taskId now ready during app ${app.id.toString} scaling, " +
```

Comment:
I think you have to add the task to `startedRunningTasks`. Or are you adding it somewhere else?